app/src/components/staking/DelegationsOverview.vue

```diff
@@ -52,17 +59,27 @@ export default {
   },
   data: () => ({
     delegations: [],
+    undelegations: [],
+    balances: [],
     delegationsLoaded: false,
+    undelegationsLoaded: false,
   }),
   computed: {
     ...mapState([`session`]),
     ...mapGetters([`address`, `currentNetwork`]),
     stakedBalance() {
-      const stakeBalance = this.balances.find(({denom}) => denom === this.currentNetwork.stakingDenom)
-      const liquidBalance =
-        Number(stakeBalance.total) - Number(stakeBalance.available)
+      const stakingDenomBalance = this.balances.find(
+        ({ denom }) => denom === this.currentNetwork.stakingDenom
+      )
+      let stakedAmount =
+        Number(stakingDenomBalance.total) -
+        Number(stakingDenomBalance.available)
+      // substract the already unbonding balance in the case of Substrate networks. There can be only one item in undelegations Array
+      if (this.undelegationsLoaded && this.undelegations.length > 0) {
+        stakedAmount -= Number(this.undelegations[0].amount)
```

Comment:
nice!

Reply:
I don't understand this simplification. This doesn't work for Cosmos am I wrong?

Reply:
are we showing this in cosmos? there is not really a need (no purgatory, no confusion from users)